zenhappens/src/styles/elements/_heroimages.scss

```
@@ -0,0 +1,118 @@
+// homepage heros
+
+.landing__hero{
+    align-items: center;
+    background-color: $color-black;
+    background-image: url("/images/zenhappens_landinghero-cropped.png");
+    background-position: 90%;
+    background-repeat: no-repeat;
+    background-size: cover;
+	display: flex;
+    flex-direction: column;
+    height: 35rem;
+    justify-content: center;
+    padding: 2em;
+    text-align: center;
+    color: #fff;
+    text-shadow: 0 0.12rem 1.3rem rgba(0,0,0,0.8);
+}
+
+.landing__text{
+    max-width: 40rem;
+}
+
+.link__button--hero{
+    border: 0.1rem solid #ffffff;
+    color: #ffffff;
+
+    &:hover, &:focus{
+        color: rgba(#ffffff,.8);
+        border: 0.1rem solid rgba(#ffffff,.8);
+      }
+}
+
+.specialoffer{
+    align-items: center;
+    background-color: $color-black;
+    background-image: url("/images/daveandrachel-full.png");
+    background-position: 25% bottom;
+    background-repeat: no-repeat;
+    background-size: 250%;
+	display: flex;
+    flex-direction: column;
+    height: 27em;
+    justify-content: center;
+    margin: $padding-mobile;
+    padding: 2em;
+    text-align: center;
+    color: #fff;
+    text-shadow: 0 0.12rem 1.3rem rgba(0,0,0,0.8);
+
+    @media screen and (min-width: 500px) {
+        background-size: cover;
+        background-position: center;
+        height: 34em;
+        margin: 2em 5vw 2em 5vw;
+    }
+
+    @media screen and (min-width: 1000px) {
+        height: 30em;
+        margin: 2em 7vw 2em 7vw;
```

You don't need to redeclare the top and bottom margins here, since they are consistent with the previous media query.
```suggestion
        margin-right: 7vw;
        margin-left: 7vw;
```
While this looks like more code (and it is!), it's preferred b/c it keeps it clear *what* changes in the media query.